docs/INSTALL.md

```diff
@@ -100,67 +93,74 @@ Here is the list of DC/OS versions supported on dcos-terraform natively:
 
 **Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
 
-To apply the configuration file, you can use this command below.
+To apply the configuration file and you accept all the default variables you can use this command below, otherwise you can continue along with this guide.
 
 ```bash
-terraform apply -var-file cluster_profile.tfvars
+terraform apply
+
 ```
 
 ## Advanced YAML Configuration
 
-We have designed this project to be flexible. Here are the example working variables that allows very deep customization by using a single `tfvars` file.
+We have designed this project to be flexible. Here are the example working variables that allows very deep customization by using a single `main.tf` file.
+
+For advanced users with stringent requirements, here are DC/OS flag examples you can simply paste in `main.tf` file.
```

Comment:
Rewrite as:
For advanced users with stringent requirements, paste the DC/OS flag examples in `main.tf` file.

Reply:
Done